sys/riscv/riscv/locore.S

```diff
@@ -107,8 +107,8 @@ _start:
 	li	t2, 512			/* Build 512 entries */
 	add	t3, t4, t2
 	li	t5, 0
-2:
 	li	t0, (PTE_KERN_CAP | PTE_X)
+2:
```

Comment:
Please do this as an upstream change that gets merged back in.

Reply:
OK; https://reviews.freebsd.org/D25524